shells/default/default.nix

```diff
@@ -41,6 +42,7 @@ let
       procps
       pstree
       flox-cli-tests
+      tre-command
```

Comment:
This doesn't appear to be used. Is it leftover from debugging?

Reply:
Yeah